api/base/renderers.py

```diff
@@ -31,3 +31,14 @@ def get_context(self, *args, **kwargs):
         for form in unwanted_forms:
             del context[form]
         return context
+
+
+class PlainTextRenderer(StaticHTMLRenderer):
+    """
+    Renders plain text.
+
+    Inherits from StaticHTMLRenderer for error handling.
+    """
+
+    media_type = 'text/markdown'
+    format = '.txt'
```

Comment:
If it affects the extension that the browser uses for downloads, it may be we want it to be `markdown` or `md`.